ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_fc.py

```diff
@@ -69,14 +69,14 @@
     - present
```

Comment:
Why is there no "absent" here? (pardon my lack of understanding how this particular API resource works)

Reply:
because u cannot delete or remove FC. it has only one api. update.not even create . hence state is always present